motion/ruby_motion_query/event.rb

```diff
@@ -32,7 +32,7 @@ def initialize(sender, event, block)
 
     def handle_gesture_or_event(gesture)
       # Don't fire :long_press events twice.
-      return if @sdk_event_or_recognizer == VIEW_GESTURES[:long_press] && gesture.state == UIGestureRecognizerStateEnded
+      return if gesture.class == VIEW_GESTURES[:long_press] && gesture.state != UIGestureRecognizerStateBegan
```

Comment:
It was a good question and I have no good answer. They do the same thing here. We can change it back to `@sdk_event_or_recognizer` if you'd like, however I think that checking the class of the var passed to the method is probably safer than checking an instance variable.